### ISSUE TITLE
Build Error on latest version - Module not found: Error: Can't resolve '#buttplug_rs_ffi_bg' 

### ISSUE THREAD

**Issue by myhubiuser1** [buttplugio/buttplug-rs-ffi#93]
**Describe the bug**
A clear and concise description of what the bug is. Remember, this should only pertain to bugs at the protocol or general architecture level. Library or application specific bugs should be files in their respective repos.
When trying to upgrade Buttplug NPM library from 1.0.15 to latest (1.0.17) I'm getting the following error when building the JS library:
Error: ./node_modules/buttplug/dist/module/buttplug-rs-ffi/buttplug_rs_ffi_bg.js
Module not found: Error: Can't resolve '#buttplug_rs_ffi_bg' in '/Users/ME/Documents/Funscript-Player/node_modules/buttplug/dist/module/buttplug-rs-ffi'
The library works from 1.0.10 up to 1.0.15

**Expected behavior**
Build to pass and run local server as older versions does. 

**Actual behavior**
Build fails with error:
Error: ./node_modules/buttplug/dist/module/buttplug-rs-ffi/buttplug_rs_ffi_bg.js
Module not found: Error: Can't resolve '#buttplug_rs_ffi_bg' in '/Users/ME/Documents/Funscript-Player/node_modules/buttplug/dist/module/buttplug-rs-ffi'

**Additional context**
Trying to make Funscript-player work with lovense toys - fork can be found here - https://github.com/myhubiuser1/Funscript-Player

**Comment by shogunsho2** [comment]
I think it is probably some error with the build pipeline. i managed to get it working by modifiying 

node_modules/dist/module/buttplug-rs-ffi/buttplug_rs_ffi.js to:

`export * from "./buttplug_rs_ffi_bg_web";`

instead of

`export * from "#buttplug_rs_ffi_bg";`

then i could at least run locally and test things out

**Comment by myhubiuser1** [comment]
Yup, that worked for me as well but definitely cannot do a proper build with it.
